Engine/CMakeLists.txt.bak

```diff
@@ -0,0 +1,33 @@
+file (GLOB_RECURSE sources CONFIGURE_DEPENDS "*.cpp" "*.h")
```

Comment:
not sure what generated this file but .bak files are not needed since we're using git.

Reply:
Not sure either. I'll delete it